scripts/build_utils.sh

```diff
@@ -616,7 +613,11 @@ setup_pbuilder() {
         # in newer versions. This ticket solves the specific issue in 8.1.1 (which vendors urllib3):
         # https://github.com/shazow/urllib3/issues/567
         echo "USENETWORK=yes" >> ~/.pbuilderrc
+        local hookdir
+        hookdir=$(recreate_hookdir)
```

Comment:
should `$()` be double quoted?  genuine question

Reply:
no need. but it is advisable to be double quoted when the variable is *referenced* and it should be considered as a single string.